src/edu/washington/escience/myria/daemon/MyriaDriverLauncher.java

```diff
@@ -229,55 +228,60 @@ private static void logStartupMessage(Class<? extends Name<?>>[] classes, Inject
       final String doc = annotation.doc();
       final String defaultVal;
       {
-        if (!annotation.default_value().equals(NamedParameter.REEF_UNINITIALIZED_VALUE))
+        if (!annotation.default_value().equals(NamedParameter.REEF_UNINITIALIZED_VALUE)) {
           defaultVal = annotation.default_value();
-        else if (!annotation.default_class().equals(Void.class))
+        } else if (!annotation.default_class().equals(Void.class)) {
           defaultVal = annotation.default_class().getSimpleName();
-        else if (annotation.default_values().length > 0)
+        } else if (annotation.default_values().length > 0) {
           defaultVal = Arrays.toString(annotation.default_values());
-        else if (annotation.default_classes().length > 0) {
-          Class<?>[] default_classes = annotation.default_classes();
-          String[] sss = new String[default_classes.length];
-          for (int i1 = 0; i1 < default_classes.length; i1++) {
-            sss[i1] = default_classes[i1].getSimpleName();
+        } else if (annotation.default_classes().length > 0) {
```

Comment:
I haven't either, TBH :) We just moved to Java 8 a few months ago and I'm trying to maintain consistency with existing style, but the REEF-specific code already uses lambdas all over the place anyway.